mdpow/fep.py

```diff
@@ -583,20 +584,9 @@ def setup(self, **kwargs):
         kwargs.setdefault('qscript', qscripts)
 
         for component, lambdas in self.lambdas.items():
-            if self.method == "TI":
-                for l in lambdas:
-                    # set up gromacs job for each FEP window in TI
-                    params = self._setup(component, l, **kwargs)
-            elif self.method == "BAR":
-                xlambdas = [None] + list(lambdas) + [None]
-                for l in lambdas:
-                    foreign_lambdas = [xlambdas[xlambdas.index(l) - 1],
-                                       xlambdas[xlambdas.index(l) + 1]]
-                    # set up gromacs job for each FEP window in BAR
-                    params = self._setup(component, l,
-                                         foreign_lambdas=foreign_lambdas, **kwargs)
-            else:
-                raise ValueError("Unknown method {}".format(self.method))
+            for l in lambdas:
```

Comment:
This works for TI, BAR, and MBAR?

Don't you have to add the end points or is it really sufficient just to repeat all lambdas?